AnalyticsKit/AnalyticsKit/AnalyticsKitTimedEventHelper.m

```diff
@@ -20,7 +20,7 @@ +(void)startTimedEventWithName:(NSString *)name forProvider:(id<AnalyticsKitProv
     [self startTimedEventWithName:name properties:nil forProvider:provider];
 }
 
-+(void)startTimedEventWithName:(NSString *)name properties:(NSDictionary *)properties forProvider:(id<AnalyticsKitProvider>)provider {
++(void)startTimedEventWithName:(NSString *)name properties:(NSDictionary *)properties forProvider:(id<AnalyticsKitProvider, NSObject>)provider {
```

Comment:
What is the `NSObejct` here doing? Looks like nothing calls these anymore. We should add a unit test for it.